libs/blocks/global-navigation/utilities/utilities.js

```diff
@@ -1,5 +1,8 @@
 import { getConfig, localizeLink } from '../../../utils/utils.js';
 
+const curtainSelector = '.feds-curtain';
+const navLinkClassName = 'feds-nav-link';
+const globalNavSelector = '.global-navigation';
```

Comment:
Can't we use the selectors from utils?

Reply:
Yes we could, but the utilities are "sync" (e.g. get loaded right away with the global-navigation)
The utils are delayed until we load the keyboard-navigation (3s or on interaction)

We can consider moving the selectors to the more general utilities file though

Reply:
Got it, no big deal, we can explore afterwards.